src/tests/multihost/ad/test_adparameters.py

```diff
@@ -474,6 +474,9 @@ def test_0013_bz1794016(self, multihost, adjoin):
          is set and files provider is also enabled
         :id: ac5e99cc-2d81-48f7-82ff-622f1c6b3684
         """
+        cmd = multihost.client[0].run_command('man sssd-files | grep files', raiseonerr=False)
```

Comment:
Please create a single helper to detect presence of 'files provider' support and use this single helper consistently in entire code base of multihost tests.

Reply:
Additionally I'd suggest to use the same approach to detect support as sssd-test-framework is using:
https://github.com/SSSD/sssd-test-framework/blob/57de0200e97fd9d0e48ec0951ac1ca3f00640edf/sssd_test_framework/hosts/client.py#L44

Reply:
@shridhargadekar @aborah-sudo @madhuriupadhye 
Those comments apply to all relevant PR, not merely to this one.

Reply:
Updated.

Reply:
We have added this method, https://github.com/SSSD/sssd/blob/master/src/tests/multihost/sssd/testlib/common/qe_class.py#L194
can you please update it to use this method.

Reply:
Updated again.